backend/app/services/dataset_item_service.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from datetime import datetime
+from io import BytesIO
+from typing import BinaryIO
+from uuid import UUID
+
+from app.schemas.dataset_item import DatasetItem
+from app.services.base import ResourceNotFoundError, ResourceType
+
+logger = logging.getLogger(__name__)
+
+
+class DatasetItemService:
```

Comment:
Renaming suggestion: `DatasetService`.

Later, it will have methods that work on individual items as well as methods that operate on the whole dataset.